common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -143,6 +145,7 @@ public static WorkbenchConfig createEmptyConfig() {
     // Controls whether all api requests are traced and sent to Stackdriver tracing, or
     // whether we only trace at the default frequency.
     public boolean traceAllRequests;
+    public String location;
```

Comment:
Follow-up note: @jaycarlton is introducing a very similar config value in https://github.com/all-of-us/workbench/pull/2978/files, but that PR uses "us-central" instead of "us-central1". I have a feeling "us-central1" is more correct since that's the GCE-aligned location name, but there may be reasons we'd need both.

You guys should sync up and make sure we don't end up with competing config values here!

Reply:
Will talk to @jaycarlton  about this, For gcoud queues we will be using us-central1 
_This location will serve as the LOCATION_ID parameter for your Cloud Tasks requests, so make a note of it. Note that two locations, called europe-west and us-central in App Engine commands, are called, respectively, europe-west1 and us-central1 in Cloud Tasks commands_
https://cloud.google.com/tasks/docs/creating-queues